client/src/assets/scss/main.scss

```diff
@@ -2,13 +2,52 @@
 @import "~foundation-sites/scss/foundation";
 @include foundation-everything;
 // place global css here
+
+body {
+  background-color: #F2E3DB;
```

Comment:
you can consider creating color variables so you don't need to keep referencing hex codes
https://sass-lang.com/documentation/variables